mezzanine_client/api.py

```diff
@@ -164,14 +164,16 @@ def get_post(self, item_id):
         """
         return self._get(['posts', int(item_id)])
 
-    def get_posts(self, offset=0, limit=10):
+    def get_posts(self, offset=0, limit=10, category="", date_min = ""):
         """
         Get published blog posts
         :param offset: pagination offset
         :param limit: pagination limit
+        :param category: the category which the post related to
+        :param limit: date_min of posts to return
```

Comment:
You copied and pasted and forgot to edit it. It should say something like `:param date_min:  minimum date published` and state the format that the date should be entered in - e.g. `(unix timestamp)` if it's using unix time.